pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -148,6 +155,28 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type Treasurer<T: Config> = StorageMap<_, Identity, SeasonId, T::AccountId, OptionQuery>;
 
+	pub struct DefaultMintTransferAllowed;
+
+	impl Get<bool> for DefaultMintTransferAllowed {
+		fn get() -> bool {
+			true
+		}
+	}
+
+	/// Flog to mark free mints as transferable by all accounts or not,
+	/// if free mint transfer is blocked, only whitelisted accounts
+	/// can still make transfers
+	#[pallet::storage]
+	pub type MintTransferAllowed<T: Config> =
```

Comment:
This should be handled in the `GlobalConfig` having one flag for general FreeMintTransfers and one for WhiteListed FreeMintTransfers

https://github.com/ajuna-network/Ajuna/blob/develop/pallets/ajuna-awesome-avatars/src/types/config.rs#L101

Add a FreeMintConfig

Reply:
Okay, updated PR!